react/Carousel.tsx

```diff
@@ -271,90 +73,272 @@ export default class Carousel extends Component<Props, State> {
     )
   }
 
-  public ArrowContainerRender: React.StatelessComponent<
-    ArrowContainerProps
-  > = ({ children }: ArrowContainerProps) => {
+  function ArrowContainerRender({ children }: ArrowContainerProps) {
     const wrapperClasses = classnames(
-      styles.arrowsContainerWrapper,
+      handles.arrowsContainerWrapper,
       'w-100 h-100 absolute left-0 top-0 flex justify-center'
     )
     const containerClasses = classnames(
       styles.arrowsContainer,
       'w-100 h-100 mw9 flex-ns justify-between items-center dn-s'
     )
-
+  
```

Comment:
```suggestion

```

Reply:
I would leave that empty line to separate the `return` statement